### ISSUE TITLE
Maintainer missing in README.md

### ISSUE THREAD

**Issue by MarcMoschSLUB** [slub/mirador-annotations-extend-plugin#1]
Please add a maintainer to the README.md

**Comment by rettinghaus** [comment]
Yes, please add a [CODEOWNERS](https://docs.github.com/de/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file.
